docs/crio.conf.5.md

```diff
@@ -145,6 +151,9 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
 
     2) `/usr/share/containers/mounts.conf`: This is the default file read for mounts. If you want CRI-O to read from a different, specific mounts file, you can change the default_mounts_file. Note, if this is done, CRI-O will only add mounts it finds in this file.
 
+**bind_mount_prefix**=""
+  A prefix to use for the source of the bind mounts.  This option would be useful if you were running CRI-O in a container.  And had / mounted on /host in your container.  Then if you ran CRI-O with the --bind-mount-prefix=/host option, CRI-O would add /host to any bind mounts it is handed over CRI.  If Kubernetes asked to have /var/lib/foobar bind mounted into the container, then CRI-O would bind mount /host/var/lib/foobar.  Since CRI-O itself is running in a container with / or the host mounted on /host, the container would end up with /var/lib/foobar from the host mounted in the container rather then /var/lib/foobar from the CRI-O container.
```

Comment:
'container.  And' to "container and '

Reply:
Maybe it's me having Friday afternoon brain, but I'm not sure what "had / mounted on /host".  Does this mean "had the '/' on the host mounted as '/host' on  your container?